src/OpenTK/opentk-1.0.projitems

```diff
@@ -0,0 +1,531 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+        <Compile Include="GlobalAssemblyInfo.cs"/>
```

Comment:
I explained that in the commit message - if we want to include 0.9 we can share the csproj if we use .projitems